blocksuite/affine/blocks/code/src/preview/renderer/html/html-render.ts

```
@@ -0,0 +1,113 @@
+  override firstUpdated(_changedProperties: PropertyValues): void {
+    const result = super.firstUpdated(_changedProperties);
+
+    this._link();
+
+    this.disposables.add(
+      this.model.props.text$.subscribe(() => {
+        this._link();
+      })
+    );
+
+    return result;
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)